chore(release): Autobump agent service version (1.0.10) (#1224)
armory/docs: content/en/armory-enterprise/release-notes/rn-armory-agent/agent-service/agent-service-v-1-0-10.md

---
title: v1.0.10 Armory Agent Service (2022-02-14)
toc_hide: true
version: 01.00.10

---

> Fixes:
> 
> - Out of bounds index while performing operations (deploy, delete, ... etc manifest)
> - Wait to the kubernetes resource to be deleted when delete manifest operation is performed; it just apply when gracePeriod is not set or greater than 0

Reviewer: ```suggestion
## Fixes

- Fixed an issue where out of bounds index errors occurred while performing operations on manifests, such as deploy or delete.
- Fixed an issue with how the Agent Service waits for Kubernetes resources to be deleted when a delete manifest operation is performed. The Agent Service now only waits until the manifest is deleted if the grace period is not set to a value greater than 0 in the UI.

```

---
title: v1.0.10 Armory Agent Service (2022-02-14)
toc_hide: true
version: 01.00.10

---

> ## Fixes
> 
> - Fixed an issue where out of bounds index errors occurred while performing operations on manifests, such as deploy or delete.
> - Fixed an issue with how the Agent Service waits for Kubernetes resources to be deleted when a delete manifest operation is performed. The Agent Service now only waits until the manifest is deleted if the grace period is not set to a value greater than 0 in the UI.
> 
> 

Reviewer: ```suggestion
## Changes
- The Agent Service now waits for Kubernetes resources to be deleted when a delete manifest operation is performed if you do not set a grace period greater than 0 in the UI.

## Fixes

- Fixed an issue where crashes occurred due to out of bounds index errors that occurred while performing operations on manifests, such as deploy or delete.
```
